### PR TITLE
 Remove highlight control when tap on touch devices #1390

### PR DESCRIPTION
**Pull request by hcodes**

### CODE REVIEW

File: design/common.blocks/link/_theme/link_theme_islands.styl [hunk 1]
```diff
@@ -47,4 +47,9 @@
             font-size: 18px;
         }
     }
+
+    &.link_disabled
+    {
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```
Comment (sipayRT) [inline design/common.blocks/link/_theme/link_theme_islands.styl:53]: can't find any highlight on tap for disabled link

Comment (sipayRT) [inline design/common.blocks/link/_theme/link_theme_islands.styl:53]: sry, already found

**Comment by sipayRT** [comment]
maybe it will be better to move these changes to the `touch` level?

**Comment by tadatuta** [comment]
> maybe it will be better to move these changes to the touch level?

we declare that desktop sites should work fine on touch devices

**Comment by sipayRT** [comment]
ok, tnx.
@hcodes  works fine for me

**Pull request merged**